src/Combinatorics/Matroids/JMatroids.jl

```diff
@@ -1,3 +1,4 @@
 include("matroids.jl")
 include("properties.jl")
 include("ChowRings.jl")
+include("realization_space.jl")
```

Comment:
```suggestion
include("realization_space.jl")

```
Please ensure every file ends with a newline